Makefile

```diff
@@ -1,5 +1,3 @@
-.PHONY: test lint autoformat run ftest install release docker-build docker-run docker-push
```

Comment:
Question: does removing all of these not break anything?

Reply:
Reference: https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile

`.PHONY` indicates that the build target is not a file, but a command. In essence `.PHONY test` says "even if there's a file called test", it should not be taken into account, but a command `test` should be executed.